pkgs/os-specific/linux/busybox/default.nix

```diff
@@ -27,35 +27,20 @@ let
 in
 
 stdenv.mkDerivation rec {
-  name = "busybox-1.27.2";
+  name = "busybox-1.28.0";
```

Comment:
All of the security patches that you removed are in this release. So nothing to worry here :+1: